Homework-Mar-7/Trie/Trie/Trie.cs

```diff
@@ -0,0 +1,153 @@
+﻿namespace Trie;
+
+/// <summary>
+/// Trie data structure implementation.
+/// </summary>
+public class Trie
+{
+    private readonly Vertex root = new ();
+
+    /// <summary>
+    /// Gets elements count.
+    /// </summary>
+    public int Size { get; private set; } = 0;
+
+    /// <summary>
+    /// Check if the element is contained.
+    /// </summary>
+    /// <param name="element">The element to check.</param>
+    /// <returns>True if the element is contained.</returns>
+    /// <exception cref="ArgumentException">Thrown if the argument is an empty string.</exception>
+    public bool Contains(string element)
+    {
+        if (element == string.Empty)
```

Comment:
Давайте на нулл еще все-таки проверять строку, вполне может прийти

```suggestion
        ArgumentException.ThrowIfNullOrEmpty(expression);
```